src/main/resources/compendium.yaml

```
@@ -0,0 +1,250 @@
+openapi: 3.0.0
+info:
+  description: Compendium is a service for schema storage, conversion, version management,
+    and client generation. It supports a RESTful interface for storing and
+    serving schemas like Avro, Protobuf, OpenAPI, and Mu.
+  version: 0.0.1
+  title: Compendium
+  contact:
+    email: contact@higherkindness.io
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+externalDocs:
+  description: Learn more about Compendium
+  url: https://github.com/higherkindness/compendium
```

That will be the official documentation site :wink: 
```suggestion
  url: https://higherkindness.io/compendium
```